packages/website/styles/markdown.css

```diff
@@ -0,0 +1,998 @@
+  line-height: 1.5;
+  word-wrap: break-word;
+}
+
+.post details {
+  display: block;
+}
+
+.post summary {
+  display: list-item;
+}
+
+.post a {
+  background-color: initial;
+}
+
+.post a:active,
+.post a:hover {
+  outline-width: 0;
+}
+
+.post strong {
+  font-weight: inherit;
+  font-weight: bolder;
+}
+
+.post h1 {
+  font-size: 2em;
+  margin: 0.67em 0;
+}
+
+.post img {
+  border-style: none;
+}
+
+.post code,
+.post kbd,
+.post pre {
+  font-family: monospace, monospace;
+  font-size: 1em;
+}
+
+.post hr {
+  box-sizing: initial;
+  height: 0;
+  overflow: visible;
+}
+
+.post input {
+  font: inherit;
+  margin: 0;
+}
+
+.post input {
+  overflow: visible;
+}
+
+.post [type='checkbox'] {
+  box-sizing: border-box;
+  padding: 0;
+}
+
+.post * {
+  box-sizing: border-box;
+}
+
+.post input {
+  font-family: inherit;
+  font-size: inherit;
+  line-height: inherit;
+}
+
+.post a {
+  color: #0366d6;
+  text-decoration: none;
+}
+
+.post a:hover {
+  text-decoration: underline;
+}
+
+.post strong {
+  font-weight: 600;
+}
+
+.post hr {
+  height: 0;
+  margin: 15px 0;
+  overflow: hidden;
+  background: transparent;
+  border: 0;
+  border-bottom: 1px solid #dfe2e5;
+}
+
+.post hr:after,
+.post hr:before {
+  display: table;
+  content: '';
+}
+
+.post hr:after {
+  clear: both;
+}
+
+.post table {
+  border-spacing: 0;
+  border-collapse: collapse;
+}
+
+.post td,
+.post th {
+  padding: 0;
+}
+
+.post details summary {
+  cursor: pointer;
+}
+
+.post kbd {
+  display: inline-block;
+  padding: 3px 5px;
+  font: 11px SFMono-Regular, Consolas, Liberation Mono, Menlo, monospace;
+  line-height: 10px;
+  color: #444d56;
+  vertical-align: middle;
+  background-color: #fafbfc;
+  border: 1px solid #d1d5da;
+  border-radius: 10px;
+  box-shadow: inset 0 -1px 0 #d1d5da;
+}
+
+.post h1,
+.post h2,
+.post h3,
+.post h4,
+.post h5,
+.post h6 {
+  margin-top: 0;
+  margin-bottom: 0;
+  font-family: 'ChicagoFLF', sans-serif;
+  font-weight: bold;
+  margin-bottom: 0.75em;
+}
+
+.post h1 {
+  font-size: 32px;
+}
+
+.post h2 {
+  font-size: 1.75em;
+}
+
+.post h3 {
+  font-size: 1.25em;
+}
+
+.post h4 {
+  font-size: 1em;
+}
+
+.post h5 {
+  font-size: 0.875em;
+}
+
+.post h6 {
+  font-size: 0.75em;
+}
+
+.post p {
+  margin-top: 0;
+  margin-bottom: 10px;
+}
+
+.post blockquote {
+  margin: 0;
+  padding: 0 1em;
+  border-left: 0.25em solid var(--nsltblue);
+}
+
+.post ol,
+.post ul {
+  padding-left: 0;
+  margin-top: 0;
+  margin-bottom: 0;
+  padding-left: 1em;
+}
+
+.post ul > li {
+  list-style-type: disc;
+  margin: 0.5rem 0;
+}
+
+.post ol > li {
+  list-style-type: decimal;
+  margin: 0.5rem 0;
+}
+
+.post ol ol,
+.post ul ol {
+  list-style-type: lower-roman;
+}
+
+.post ol ol ol,
+.post ol ul ol,
+.post ul ol ol,
+.post ul ul ol {
+  list-style-type: lower-alpha;
+}
+
+.post dd {
+  margin-left: 0;
+}
+
+.post code,
+.post pre {
+  font-family: SFMono-Regular, Consolas, Liberation Mono, Menlo, monospace;
+  font-size: 12px;
+}
```

Comment:
`code` and `pre` are already defined a few lines up can we merge these ?

Also can you make a var for `SFMono-Regular, Consolas, Liberation Mono, Menlo, monospace;` and use it everywhere we want monospace ?